xarray/backends/zarr.py

```diff
@@ -106,6 +106,24 @@ def __getitem__(self, key):
         # could possibly have a work-around for 0d data here
 
 
+def _squeeze_var_chunks(var_chunks, name=None):
```

Comment:
Can we get a quick comment to explain what this function does?

Reply:
https://github.com/pydata/xarray/pull/8253/commits/818ba67dbe7af94d7d4ced49e461e8c6e23482e8